handbook/information-security/risk-management.md

```diff
@@ -0,0 +1,22 @@
+## Risk Management
+
+### Risk Register
+
+The company [Risk
+Register](https://docs.google.com/spreadsheets/d/1gyK0zWSVlX-ZnXsN_c4-my86-hHjHEDY6IEP5h9UuVQ/edit?usp=sharing)
+tracks all of the risks that members of the DVELP team have identified.
+
+Where the anticipated impact of an unmitigated risk exceeds a rating of 4, the
```

Comment:
Can we detail the scale in this document?

Reply:
Now added